Update direct-routing-configure.md (#2987)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/direct-routing-configure.md

|Required?|Name|Description|Default|Possible values|Type and restrictions|
|:-----|:-----|:-----|:-----|:-----|:-----|
|Yes|FQDN|The FQDN name of the SBC |None|NoneFQDN name, limit 63 characters|String, list of allowed and disallowed characters on [Naming conventions in Active Directory for computers, domains, sites, and OUs](https://support.microsoft.com/help/909264)|

> |No|MediaBypass |Parameter indicated of the SBC supports Media Bypass and the administrator wants to use it.|None|True<br/>False|Boolean|

Reviewer: ```suggestion
|No|MediaBypass |Parameter indicated of the SBC supports Media Bypass and is for use by the administrator.|None|True<br/>False|Boolean|
```
